x-pack/plugin/sql/qa/src/main/resources/docs.csv-spec

```diff
@@ -1900,3 +1902,59 @@ SELECT WEEK(CAST('1988-01-05T09:22:10Z' AS TIMESTAMP)) AS week, ISOWEEK(CAST('19
 2              |1
 // end::weekOfYear
 ;
+
+
+currentNow
+// tag::filterNow
+SELECT first_name FROM emp WHERE hire_date > NOW() - INTERVAL 100 YEARS ORDER BY first_name ASC LIMIT 5;
+
+  first_name   
+---------------
+Alejandro      
+Amabile        
+Anneke         
+Anoosh         
+Arumugam       
+// end::filterNow
+;
+
+currentTimestamp-Ignore
```

Comment:
why is this and the tests below ignored?

Reply:
Because the file is static while the value of now changes on each call and thus the assertion will fail. Yet it is defined here so it can be included in the docs.

Reply:
again, I feel stupid :-) Thx!